lib/build.js

```diff
@@ -2,7 +2,13 @@ const path = require("path")
 const findUp = require("find-up")
 const { init } = require("./init")
 
-const build = async ({ dir, verbose, skipNextBuild }) => {
+const build = async ({
+  dir,
+  verbose,
+  skipBuild,
+  nextless,
+  onlyApiFiles = false,
```

Comment:
Should we add defaults for all of these bools?
```suggestion
  skipBuild = false,
  nextless= false,
  onlyApiFiles = false,
```